reference/sql/data-types/date-and-time.md

```diff
@@ -77,7 +79,13 @@ Different types of zero value are shown in the following table:
 
 Invalid `DATE`, `DATETIME`, `TIMESTAMP` values are automatically converted to the corresponding type of zero value ( '0000-00-00' or '0000-00-00 00:00:00' ) if the SQL mode permits such usage.
 
+<<<<<<< HEAD
```

Comment:
@toutdesuite Please resolve the conflict in this PR.

Reply:
Conflict resolved. PTAL~